src/core/tranche/operator.sol

```diff
@@ -26,18 +26,13 @@ contract ReserveLike{
     function borrow(address, uint) public;
 }
 
-contract QuantLike {
-   function updateDebt(int) public;
-   function updateBorrowRate() public;
-}
-
 contract SlicerLike {
    function getSlice(uint) public returns(uint);
    function getPayout(uint) public returns(uint);
 }
 
 // Operator
-// Manages the reserve. Triggers supplyRate & borrowRate calculations.
+// Interface of a tranche. Coordinates investments and borrows to/from the tranche.
 contract Operator is DSNote {
```

Comment:
If we don't use the note modifier anywhere, we shouldn't extend from DSNote

Reply:
isn't it used with all the public functions